### PR TITLE
Add table listing trade-offs

### PR DESCRIPTION
**Pull request by mogul**
Be a little more explicit here.
Ref https://twitter.com/benbravo73/status/781125385777999872

### CODE REVIEW

File: content/apps/experimental/docker.md [hunk 1]
```diff
@@ -19,7 +19,14 @@ To push a Docker image, use the `-o` flag when pushing your app, for example:
...
+Once you push a Docker image as your application, cloud.gov cannot update the baseline for your application. You are responsible for keeping it up to date. Please see [this description of responsibilities]({{< relref "intro/technology/responsibilities.md">}}). However, please note that by using docker images **you are accepting responsibility** to maintain the operating system, libraries, application code, and all the associated configurations and losing some of the benefits of a PaaS environment.
+
+Here are some considerations to keep in mind when deciding to use Docker images instead of buildpacks in your application's deployment:
+
+|   | Supported Buildpack | Docker Container  | 
+|---|---|---|
+| Pros | It "just works"<br />Automatic and constant security updates<br \>All you need to do is write code | Can build container images and run containers on local workstation<br \>Fine-grained control over compilation and root filesystem |
+| Cons | Difficult to recreate the execution environment locally<br \>Testing compilation and the result of staging is harder | Added responsibility for all security updates and bug fixes<br \>More compliance responsibility => more work  |
```
Comment (afeld) [inline content/apps/experimental/docker.md:29]: Mind citing the tweet/author/deck here?

**Review by afeld — CHANGES_REQUESTED** [review]

File: content/apps/experimental/docker.md (hunk 1 above)
Comment (mogul) [reply content/apps/experimental/docker.md:29]: No, but I don't know how best to do it. Is there a norm?

Comment (afeld) [reply content/apps/experimental/docker.md:29]: I'd say "table inspired by such-and-such" or "more information" in italics below the table like a caption, or you could even put it in an HTML comment if you didn't want it to distract readers.

Comment (mogul) [reply content/apps/experimental/docker.md:29]: The context was pretty free of additional context; it was a photo of a slide being shown on stage at the CF Summit in Europe this week. I'll put it in as an HTML comment.

**Comment by mogul** [comment]
(It's unclear how to explicitly indicate that I addressed the requested feedback and ask for verification.)

**Pull request merged**